### PR TITLE
Slider Responsiveness

### PR DESCRIPTION
**Pull request by smt96700**
Adjusted the responsive behavior of the slider component to improve usability on mobile devices.

### CODE REVIEW

**Comment by kingsmen732** [comment]
Can you please use the latest version from the repo and include your feature request  @smt96700

**Comment by smt96700** [comment]
I have updated my feature request and ensured compatibility with the latest version of the repository; please review the changes and feel free to suggest any improvements.

**Review by kingsmen732 — CHANGES_REQUESTED** [review]
Kindly look into the project section close buttons. Your feature request is not responding as it is tampered with in your base. Rest assured, other features are working pretty cool.

File: src/components/modals/ProjectModal.jsx [hunk 1]
```diff
@@ -19,7 +20,7 @@ const ProjectModal = (props) => {
           src={cross}
           alt="Close"
           className="absolute top-5 right-5 w-6 h-6 cursor-pointer hover:scale-110 transition-transform duration-200 ease-in-out"
-          onClick={() => props?.setIsModalOpen(false)}
+          onClick={() => props?.onClose()}
```
Comment (smt96700) [inline src/components/modals/ProjectModal.jsx:23]: I have replaced incorrect setIsModalOpen  with onClose  function to resolve modal close functionality.

Comment (kingsmen732) [reply src/components/modals/ProjectModal.jsx:23]: Thanks, There is some issue of touch sensitivity in mobile view that they cannot scroll down and if they try the page changes.

Comment (smt96700) [reply src/components/modals/ProjectModal.jsx:23]: Got it will review it soon.

Comment (smt96700) [reply src/components/modals/ProjectModal.jsx:23]: I've refined the slider's responsiveness for an enhanced mobile experience. Please review the changes.

**Comment by smt96700** [comment]
Could you please review the pr I have updated my local branch with the current version of the main branch.

**Pull request merged**